hadoop-hdds/server-scm/src/main/resources/webapps/scm/scm-overview.html

```diff
@@ -153,6 +155,35 @@ <h2>Meta-Data Volume Information</h2>
     </tbody>
 </table>
 
+<h2>SCM Roles (HA)</h2>
+<h4 ng-show="$ctrl.overview.jmx.RatisRoles.length == 1 && $ctrl.overview.jmx.RatisRoles[0].length == 1">{{$ctrl.overview.jmx.RatisRoles[0][0]}}</h4>
```

Comment:
This part seems to be taken from https://github.com/apache/ozone/pull/5311, but I don't think this applies here. From my understanding this is to handle when there are exceptions when calling `OzoneManager#getRatisRoles`, but `StorageContainerManager#getRatisRoles` do not throw such exceptions. It might be safe to remove this for SCM since it doesn't throw exceptions.

I noticed there are some differences in the implementation of `StorageContainerManager#getRatisRoles` and `OzoneManager#getRatisRoles`. We might need to unify the logics.

Reply:
@ivandika3 @dombizita

Thank you very much for reviewing the code! I carefully readed both the SCM and OM code, and found that integrating `OzoneManager#getRatisRoles` with `StorageContainerManager#getRatisRoles` is not very straightforward. 

However, for the sake of code readability, I adjusted the `StorageContainerManager#getRatisRoles` method to follow the implementation style of OM. 

Could you review it again and provide any suggestions?

Reply:
> I carefully readed both the SCM and OM code, and found that integrating OzoneManager#getRatisRoles with StorageContainerManager#getRatisRoles is not very straightforward.

Thanks for addressing the review. I think the latest change is fine, there is no need to completely reduce duplication, standardizing the implementation should be fine.

Now that `getRatisRoles` returns an exception in case of some exception, I think this line does not need to be removed.

`<h4 ng-show="$ctrl.overview.jmx.RatisRoles.length == 1 && $ctrl.overview.jmx.RatisRoles[0].length == 1">{{$ctrl.overview.jmx.RatisRoles[0][0]}}</h4>`

This line will print the exception in case the SCM service is a STANDALONE cluster.

Reply:
Thank you very much for your comment! I will continue to improve the code.